config/settings/base.py

```diff
@@ -30,6 +30,9 @@
 # SECURITY WARNING: keep the encryption key used in production secret!
 ENCRYPTION_KEY = os.environ.get("ENCRYPTION_KEY")
 
+# X Frame Options
+X_FRAME_OPTIONS = "DENY"
+
```

Comment:
Why was the choice made to deny the framing?

Reply:
This is the default value (that I specifically reiterated here)